src/tracks/ui/BrushHandle.cpp

```diff
@@ -282,31 +285,44 @@ UIHandle::Result BrushHandle::Drag
    mMostRecentX = x;
    mMostRecentY = y;
 
+   double posTime;
+   wxInt64 posFreq;
+   auto posToLongLong = [&](int x0){
+      posTime = viewInfo.PositionToTime(x0, mRect.x);
+      sampleCount sc = wt->TimeToLongSamples(posTime);
+      return sc.as_long_long();
+   };
+
    // Clip the coordinates
    // TODO: Find ways to access the ClipParameters (for the mid)
-   int x1 = std::max(mRect.x + 10, std::min(x, mRect.x + mRect.width - 20));
-   int y1 = std::max(mRect.y + 10, std::min(y, mRect.y + mRect.height - 10));
+   int dest_xcoord = std::max(mRect.x + 10, std::min(x, mRect.x + mRect.width - 20));
+   int dest_ycoord = std::max(mRect.y + 10, std::min(y, mRect.y + mRect.height - 10));
```

Comment:
Why are you excluding certain extreme values of x and y coordinates?  Is that also something made unnecesary by bounds checkin somewhere else?